RetailCoder.VBE/Inspections/ImplicitByRefParameterInspection.cs

```diff
@@ -24,8 +24,11 @@ public override IEnumerable<InspectionResultBase> GetInspectionResults()
             var interfaceMembers = UserDeclarations.FindInterfaceImplementationMembers();
 
             var issues = (from item in UserDeclarations
-                where !item.IsInspectionDisabled(AnnotationName)
+                where
+                    !item.IsInspectionDisabled(AnnotationName)
                     && item.DeclarationType == DeclarationType.Parameter
+                    // ParamArray parameters do not allow an explicit "ByRef" parameter mechanism.               
```

Comment:
Now that I think about it, it makes total sense.

> How do I pass a parameter?
>
> - ByRef
> - ByVal
> - ParamArray

The `ParamArray` modifier would be grammatically at the same "level" as `ByRef`/`ByVal` modifiers - hence it makes sense that it can't be combined with another.